### ISSUE TITLE
[traceback] ImportError: cannot import name 'get_scenarios'

### ISSUE THREAD

**Issue by stephendonner** [stephendonner/stubattribution-loadtests#5]
When running:
```
molotov -cx -d 1 loadtest.py
```

using molotov 0.2, I now see (https://gist.github.com/stephendonner/10f62588df81c315ebc589b4d90dfba0)

```
Successfully installed molotov-0.2
You are using pip version 7.1.2, however version 9.0.1 is available.
(stubattribution-loadtests) sdonner-17447:stubattribution-loadtests sdonner$ molotov -cx -d 1 loadtest
loadtest.py          loadtest_output.txt  
(stubattribution-loadtests) sdonner-17447:stubattribution-loadtests sdonner$ molotov -cx -d 1 loadtest.py
Traceback (most recent call last):
  File "/Users/sdonner/.pyenv/versions/stubattribution-loadtests/bin/molotov", line 9, in <module>
    load_entry_point('molotov==0.2', 'console_scripts', 'molotov')()
  File "/Users/sdonner/.pyenv/versions/3.5.0/envs/stubattribution-loadtests/lib/python3.5/site-packages/pkg_resources/__init__.py", line 558, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/Users/sdonner/.pyenv/versions/3.5.0/envs/stubattribution-loadtests/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2682, in load_entry_point
    return ep.load()
  File "/Users/sdonner/.pyenv/versions/3.5.0/envs/stubattribution-loadtests/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2355, in load
    return self.resolve()
  File "/Users/sdonner/.pyenv/versions/3.5.0/envs/stubattribution-loadtests/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2361, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Users/sdonner/.pyenv/versions/3.5.0/envs/stubattribution-loadtests/lib/python3.5/site-packages/molotov/run.py", line 8, in <module>
    from molotov.fmwk import runner, get_scenarios
ImportError: cannot import name 'get_scenarios'
```

**Comment by stephendonner** [comment]
That's with this revision of loadtest.py: https://github.com/stephendonner/stubattribution-loadtests/blob/5aff2c1a3186cc8853ba8149bd37508867ac0472/loadtest.py

**Comment by stephendonner** [comment]
Mitigated this for now by pinning molotov to 0.1 in requirements.txt: https://github.com/stephendonner/stubattribution-loadtests/commit/02997b3fc3902e3be63afb25252e84d02b7e3dc1

**Comment by stephendonner** [comment]
This was fixed with molotov==0.3; see output in https://github.com/loads/molotov/issues/18#issuecomment-275548501
